client/gutenberg/extensions/presets/jetpack/utils/get-extensions.js

```diff
@@ -0,0 +1,29 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import extensionSlugsJson from '../index.json';
+import { isEnabled } from 'config';
+
+const extensionSlugs = [
+	...extensionSlugsJson.production,
+	...( isEnabled( 'jetpack/blocks/beta' ) ? extensionSlugsJson.beta : [] ),
+];
+
+export default async function getExtensions() {
+	const promises = extensionSlugs.map( slug =>
+		// Need to explicitly look for `index.js` or Webpack will try with
+		// all files when resolving the dynamic import -- including `README.md`s,
+		// leading to warnings during startup.
+		import( /* webpackMode: "eager" */ `../../../${ slug }/index.js` ).then(
```

Comment:
this will pull in everything under `../../../` to the chunk, including any beta slugs. Is that ok?